packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -149,10 +155,15 @@ class AppBar extends StatelessWidget {
 
   double get _toolBarHeight => kToolBarHeight;
 
-  /// The height of this app bar when fully expanded.
+  /// By default: the height of the toolbar and the tabbar (if any).
```

Comment:
I think it's ok to say that (though I'd say it in long form, as in, "By default, this will return...") but it doesn't make sense to start the definition with that. First you have to say what it is, so that the reader has the context to know what the default means.